torch/csrc/jit/passes/frozen_graph_optimizations.h

```diff
@@ -13,6 +13,10 @@
 namespace torch {
 namespace jit {
 
+bool nonConstantParameters(Node* n);
```

Comment:
nit: could we move this to something like `torch/csrc/jit/passes/optimization_utils` ?